schema.prisma

```diff
@@ -522,6 +525,48 @@ model BankTransactionsFile {
   @@map("bank_transactions_files")
 }
 
+model CampaignReport {
+  id                  String               @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
+  campaignId          String               @map("campaign_id") @db.Uuid
+  campaign            Campaign             @relation(fields: [campaignId], references: [id])
+  startDate           DateTime             @map("start_date")
+  endDate             DateTime             @map("end_date")
+  description         String
+  files               CampaignReportFile[]
+  totalFunds          Int?               @map("total_funds") @default(0)
+  fundsForPeriod      Int?               @map("funds_for_period") @default(0)
+  spentFundsForPeriod Int?               @map("spent_funds_for_period") @default(0)
+  goals               String
+  nextSteps           String               @map("next_steps")
+  additionalInfo      String               @map("additional_info")
+  creatorId           String               @map("creator_id") @db.Uuid
+  creator             Person               @relation(fields: [creatorId], references: [id])
```

Comment:
let's unify the naming convention:
creator could be createdBy or uploadedBy

This will allow in the next iterations to add columns for 'approvedBy' or 'deletedBy'. Similarly we have the need to add some timestamps like 'createdAt, deletedAt to record when such actions happened